### PR TITLE
Bump version to 2.0.0

### PR DESCRIPTION
**Pull request by mrocklin**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mrocklin** [comment]
@conda-forge-admin, please rerender

**Comment by mrocklin** [comment]
> I tried to re-render for you but ran into some issues, please ping conda-forge/core for further assistance.

*Lights the @jakirkham Bat Signal*

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,11 +16,11 @@ build:
 
 requirements:
   build:
-    - python
+    - python >= 3.5
     - pip
 
   run:
-    - python
+    - python= 3.5
```
Comment (jakirkham) [inline recipe/meta.yaml]:
```suggestion
    - python >= 3.5
```

**Comment by jakirkham** [comment]
Looks good. One comment above.

Also, for reference, here's [the noarch python docs]( http://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python ).

**Comment by TomAugspurger** [comment]
@jakirkham was https://github.com/conda-forge/dask-core-feedstock/pull/41#issuecomment-505560348 a request to remove noarch? I took that as a preference for

```
- python >= 3.5
```

rather than a `skip: True  # [py2k]`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,20 +7,19 @@ package:
...
 
 build:
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
-  noarch: python
 
 requirements:
   build:
-    - python
+    - python >= 3.5
```
Comment (TomAugspurger) [inline recipe/meta.yaml]: No space here I think. And on line 22

```
- python >=3.5
```

**Comment by jakirkham** [comment]
Sorry for the confusion. `noarch` can stay here and the `skip` dropped.

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Oops, sorry for the extra space above. Thanks for catching that Tom. 🙂

**Comment by mrocklin** [comment]
I'm not seeing any skip, but I'm happy to leave things as they are :)

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
@TomAugspurger, would you like to give it another look?

**Pull request merged**